core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/BaseResourceCreateRequest.java

```diff
@@ -298,7 +314,7 @@ protected ResourceDocument.Builder completeResource(final ResourceDocument.Build
 		return builder;
 	}
 	
-	private ResourceDocument createResourceDocument(TransactionContext context, List<String> bundleAncestorIds) {
+	private ResourceDocument createResourceDocument(TransactionContext context, List<String> collectionAncestorIds) {
```

Comment:
I'm a bit hesitant with the naming here, this is an abstract resource class and if I look at it, it makes me feel that a resource must have a collection parent, but not a bundle. Whereas there could be both. What about renaming the parameter to something like `containerAncestorIds` or `parentContainers` and adding a comment next to the builder method `bundleAncestorIds` of its meaning, and the fact that it was not renamed due to avoiding schema changes?

Reply:
Agree. Will take care of it next week.